manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -222,6 +224,9 @@ jobs:
       - name: cf
     update:
       serial: true
+    properties:
+      bbs:
+          url: (( concat "https://" meta.bbs.api_location "/v1/actual_lrp_groups/list" ))
```

Comment:
Note (partly to self): I've tested this change from alphagov/paas-datadog-for-cloudfoundry-boshrelease#16 and it works. We'll need to update this PR with a new final release when merged.